core/src/main/resources/lib/form/file.jelly

```diff
@@ -23,29 +23,32 @@ THE SOFTWARE.
 -->
 
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form">
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form"
+         xmlns:m="jelly:hudson.util.jelly.MorphTagLibrary">
   <st:documentation>
     Generates an input field <code>&lt;input type="file" ... /></code>
+    All unknown attributes are passed through to the field.
 
     <st:attribute name="field">
       Used for databinding.
     </st:attribute>
-    <st:attribute name="jsonAware">
-      Enable structured form submission.
+    <st:attribute name="clazz">
+      Additional CSS class(es) to add.
     </st:attribute>
```

Comment:
Even though no longer explicit in the code, this is the only element this applies to per https://github.com/jenkinsci/jenkins/blob/c3e29a4398adf625600746f99989d18570608fc6/war/src/main/webapp/scripts/hudson-behavior.js#L2422 and so this change removes the only documentation there is for it.

(If instead support is dropped, it should be removed from callers and `hudson-behavior.js` as well.)